src/Templates/MethodTemplate.cshtml

```diff
@@ -42,8 +42,8 @@ func (client @(Model.Owner)) @(Model.Name)(@Model.MethodParametersSignature()) (
 @if (Model.IsLongRunningOperation())
 {
     <text>
-        if result.Response() != nil {
-        sc = result.Response().StatusCode
+        if result.FutureAPI != nil && result.FutureAPI.Response() != nil {
+        sc = result.FutureAPI.Response().StatusCode
```

Comment:
These two lines are the only thing I changed in this PR.
Other changes are here because the VS is always auto-format this thing which is quite annoying